ConversationsApp/Models/CoreData/PersistentParticipantDataItem+CoreDataExt.swift

```diff
@@ -34,8 +34,9 @@ extension PersistentParticipantDataItem {
             self.sid = participant.sid
             self.conversationSid = conversationSid
             self.identity = participant.identity
-            self.type = Int16(participant.type.rawValue)
-            
+            // TODO: Properly convert this channelType to a string, currently it is just a Swift enum and the channel string is hidden for Swift.
+            self.channel = String(describing: participant.channelType)
```

Comment:
Why don't use just [Participant.channel](https://sdk.twilio.com/ios/conversations/releases/4.0.3/docs/Classes/TCHParticipant.html#//api/name/channel) here?

Reply:
That variable is not visible in the Swift Context, I think the target is missing the bridging-header to automate that translation between Objc & Swift.

![Screenshot 2024-06-13 at 5 57 42 PM](https://github.com/twilio/twilio-conversations-demo-ios-swift/assets/4898461/376d76cf-0450-4e42-93d6-071b03beee56)